File: 0125-valid-palindrome/0125-valid-palindrome.py
class Solution(object):
    def isPalindrome(self, s):
        words = s.split()
        sen = "".join(words)
        sen1 = []
        for x in sen:
            if x.isalnum():
               sen1.append(x)
        sen2 = "".join(sen1)
        sen3 = sen2.lower()
        rev = sen3[::-1]
        if rev == sen3:
            return True
        else:
            return False
        
        
        
        

File: 0125-valid-palindrome/0125-valid-palindrome.cpp
class Solution {
public:
    bool isPalindrome(string s) {
        string st;
        for(char x:s)
        {
            if(isalnum(x))
            {
                st.push_back(tolower(x));
            }
        }
        int i=0;
        int j=st.size()-1;
        
        while(i<j)
        {
            if(st[i] != st[j])
            {
                return false;
                
            }
            i++;
            j--;
        }
        return true;
    }
};